surf-auth/surfAuthDebot.sol

```diff
@@ -0,0 +1,128 @@
+pragma ton-solidity >=0.35.0;
+pragma AbiHeader expire;
+pragma AbiHeader time;
+pragma AbiHeader pubkey;
+
+import "../Debot.sol";
+import "../Terminal.sol";
+import "../Menu.sol";
+import "../AddressInput.sol";
+import "../ConfirmInput.sol";
+import "../Upgradable.sol";
+import "../Sdk.sol";
+import "./Network.sol";
+import "./Base64.sol";
+import "./SigningBoxInput.sol";
+import "./UserInfo.sol";
+
+
+contract TodoDebot is Debot, Upgradable {
+    string m_id;
+    string m_pin;
+    string m_otp;
+    string m_callbackUrl;
+    uint32 m_sigingBoxHandle;
+    uint256 m_userPK;
+
+    bytes m_icon;
+
+    function start() public override { }
```

Comment:
maybe we need some output like
Terminal.print(0, "DeBot doesn't have interactive flow");
?

Reply:
Updated 'hello" field in DebotInfo exactly for that case.